cmd/stack_create_test.go

```diff
@@ -214,3 +214,36 @@ func TestStackAlreadyExists(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestStackCreateMissingArguments(t *testing.T) {
+	args := []string{"stack", "create"}
+	_, err := cmdtest.RunAppsodyCmd(args, ".", t)
+
+	if err == nil {
+		t.Fatal(err)
```

Comment:
Related to the above comment, but there isn't much point in returning the err when it's been confirmed to be `nil`